Documentation/configuration/verify-image-signatures.rst

```diff
@@ -26,14 +26,13 @@ Let's verify a Cilium image's signature using the ``cosign verify`` command:
 
 .. code-block:: shell-session
 
-    $ COSIGN_EXPERIMENTAL=1 cosign verify --certificate-github-workflow-repository cilium/cilium --certificate-oidc-issuer https://token.actions.githubusercontent.com --certificate-github-workflow-name "Image Release Build" --certificate-github-workflow-ref refs/tags/[RELEASE TAG] quay.io/cilium/cilium:v1.13 | jq
+    $ cosign verify --certificate-github-workflow-repository cilium/cilium --certificate-oidc-issuer https://token.actions.githubusercontent.com --certificate-github-workflow-name "Image Release Build" --certificate-github-workflow-ref refs/tags/[RELEASE TAG] quay.io/cilium/cilium:v1.13 --certificate-identity "https://github.com/cilium/cilium/.github/workflows/build-images-releases.yaml@refs/tags/v1.13.0" | jq
     
 
 .. note::
 
-    ``COSIGN_EXPERIMENTAL=1`` is used to allow verification of images signed in 
-    ``KEYLESS`` mode. To learn more about keyless signing, please refer to 
-    `Keyless Signatures`_.
+    ``cosign`` is using verifying images signed in ``KEYLESS`` mode. To learn
```

Comment:
` ``cosign`` is using verifying images signed in ``KEYLESS`` mode.`

-> Should this be `... is used to verify images...` instead?

(Hi Jed! :wave:)

Reply:
Hi, Quentin! Thanks for the feedback =)